scripts.js

```diff
@@ -83,7 +89,7 @@ function findIdeaByID(id) {
 }
 
 $("#ideas").on('click', "#up-btn", function(){
-  var id = +$(this).parent().attr('id');
+  var id = +$(this).closest("article").attr('id');
```

Comment:
@DanGrund can we change +$(this).closest("article").attr('id') to +$(this).closest("article").id ?